product_variant_configurator/models/product_configurator.py

```diff
@@ -185,8 +185,15 @@ def unlink(self):
             attributes.unlink()
         return result
 
+    @api.multi
+    def check_configuration_validity(self):
+        for rec in self:
+            if any(not x.value_id for x in rec.product_attribute_ids):
+                raise exceptions.ValidationError(
+                    _("You have to fill all the attributes values."))
+
     @api.model
-    def check_configuration_validity(self, vals):
+    def check_configuration_validity_from_vals(self, vals):
```

Comment:
Let's remove dead code, it can be reintroduced later if needed! Actually check_configurator_validity should not be needed as there is already the same check in https://github.com/OCA/product-variant/blob/cc88b0ce71bf907c4f300ef87f0cc03e59083c2f/product_variant_configurator/models/product_product.py#L81  so please revert this commit.